src/bsk201903/connectMultipleClusters.md

```diff
@@ -197,15 +224,15 @@ In other news, you end up having one extra Pod that doesn't do much — it's jus
 
 But the benefit is that you didn't have to write any new resource to make your Deployments federated.
 
-Every resource that creates Pod is automatically federation-ready.
+**Every resource that creates Pod is automatically federation-ready.**
 
 It's a neat idea because you can suddenly have multi-region clusters without even noticing it, but it also feels quite risky as there's a lot of magic going on.
```

Comment:
Devil's advocate. Shipper doesn't do federated ingress either unless I'm missing something?

Reply:
the honestbee example wasn't really applicable to the thread because kubefed v1 didn't have ingress federation

and the issue we were trying to address is handled by neither shipper or multi-cluster-scheduler

just that the "magic" aspect of having things propagated across regions by the multi-cluster-scheduler only applies to pods and nothing else - felt like the title referring to "magic federation" is misleading

Reply:
I double checked that (after I made that comment) and multi-cluster-scheduler offers federated Services too. (Which it makes sense since it has to provide a consistent Service layer across clusters). Don't you still believe in magic ?! :P

> The service reroute controller modifies services whose endpoints target proxy pods. The keys of their label selectors are prefixed with multicluster.admiralty.io/, to match corresponding delegate pods (see below). Also, the services are annotated with io.cilium/global-service=true, to be load-balanced across a Cilium cluster mesh.